agent/api_manager/osv_service_api.py

```diff
@@ -0,0 +1,194 @@
+import dataclasses
+import json
+import logging
+from typing import Optional, Iterator
+
+import requests
+from ostorlab.agent.kb import kb
+from ostorlab.agent.mixins import agent_report_vulnerability_mixin
+
+from agent import cve_service_api
+from agent import osv_output_handler
+
+logger = logging.getLogger(__name__)
+
+OSV_ENDPOINT = "https://api.osv.dev/v1/query"
+
+RISK_RATING_MAPPING = {
+    "POTENTIALLY": agent_report_vulnerability_mixin.RiskRating.POTENTIALLY,
+    "LOW": agent_report_vulnerability_mixin.RiskRating.LOW,
+    "MEDIUM": agent_report_vulnerability_mixin.RiskRating.MEDIUM,
+    "MODERATE": agent_report_vulnerability_mixin.RiskRating.MEDIUM,
+    "HIGH": agent_report_vulnerability_mixin.RiskRating.HIGH,
+    "CRITICAL": agent_report_vulnerability_mixin.RiskRating.CRITICAL,
+}
```

Comment:
You don't need a map, just use `agent_report_vulnerability_mixin.RiskRating['XXXX']`

Reply:
+1